### PR TITLE
[MAJOR] Implement system testing #15

### PR DESCRIPTION
**Pull request by bowsong**
Installed cypress and wrote a basic test using the stock reactjs page

### CODE REVIEW

**Review by nasaku898 — DISMISSED** [review]
Looks good and good example to get us started with cypress

**Comment by nasaku898** [comment]
And change the title of the PR from [EPIC] to [MAJOR]

**Comment by Manraj45** [comment]
Why is there a mp4 file?

**Comment by bowsong** [comment]
> Why is there a mp4 file?

Cypress makes videos from the test it runs

**Comment by bowsong** [comment]
We could add that folder in the gitignore so it doesn't pollute our repo

**Comment by nasaku898** [comment]
> We could add that folder in the gitignore so it doesn't pollute our repo

yeah I think it's a good idea. Could you add it to the gitignore.

**Review by nasaku898 — APPROVED** [review]
Looks good, it was good idea to add the mp4 to the gitignore!

**Review by Manraj45 — APPROVED** [review]

File: frontend/cypress.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
[the remark this answers is not in the document]
Comment (bowsong) [reply frontend/cypress.json:1]: Yeah, but there wasn't any configs i could think of adding

**Pull request merged**